<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始捕捉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>停止捕捉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pre_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>输出信息：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>直播间房号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomlineE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="path_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_lineE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="list_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>直播列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
